coalib/bearlib/languages/documentation/DocumentationExtraction.py

```diff
@@ -214,9 +214,8 @@ def extract_documentation_with_markers(content, docstyle_definition):
     :param content: The source-code-string where to extract documentation from.
                     Needs to be a list or tuple where each string item is a
                     single line (including ending whitespaces like ``\\n``).
-    :param markers: The list/tuple of marker-sets that identify a
-                    documentation-comment. Low-index markers have higher
-                    priority than high-index markers.
+    :param docstyle_definition: The ``DocstyleDefinition`` instance that
```

Comment:
The param descriptions should be consistently aligned, like in `extract_documentation` below. If you feel that there is not enough space left, just add line breaks like in https://github.com/coala/coala/pull/4015